src/libsyntax/feature_gate.rs

```diff
@@ -686,6 +683,9 @@ declare_features! (
     (accepted, extern_crate_item_prelude, "1.31.0", Some(55599), None),
     // Allows use of the `:literal` macro fragment specifier (RFC 1576).
     (accepted, macro_literal_matcher, "1.31.0", Some(35625), None),
+    // Allows irrefutable patterns in if-let and while-let statements (RFC 2086)
+    (accepted, irrefutable_let_patterns, "1.32.0", Some(44495), None),
```

Comment:
```suggestion
    (accepted, irrefutable_let_patterns, "1.33.0", Some(44495), None),
```
It slipped so you'll have to bump the version.